### PR TITLE
WDL value head

### PR DESCRIPTION
**Pull request by Cyanogenoid**
This implements a Win/Draw/Loss value head with cross-entropy loss instead of the current scoring of wins as 1, draws as 0, and loss as -1 with an MSE loss. In early testing, it seems to be working.

MSE loss is no longer used in optimisation, but is still reported for comparison by averaging [1, 0, -1] weighted by the win, draw, and loss probabilities.

As this is a change in the architecture, merging this requires support in lc0.

### CODE REVIEW

**Comment by Cyanogenoid** [comment]
Interestingly, on the CCRL dataset, the final MSE loss is exactly the same as the baseline model that minimises MSE rather than cross-entropy on the value head. Policy loss is the same too. The value loss steadily goes down as you would expect.

**Comment by Cyanogenoid** [comment]
I've renamed Accuracy to Policy Accuracy in Tensorboard for consistency in order to add Value Accuracy. Old logs are still compatible, they just still show up on Accuracy and thus are not as easily comparable.

File: tf/tfprocess.py [hunk 1]
```diff
@@ -521,10 +543,10 @@ def construct_net(self, planes):
         self.weights.append(W_fc2)
         self.weights.append(b_fc2)
         h_fc2 = tf.nn.relu(tf.add(tf.matmul(h_conv_val_flat, W_fc2), b_fc2))
```
Comment (Tilps) [inline tf/tfprocess.py]: I am wondering if this change should be accompanied by an increase in fc2 outputs to 256.  256 seems a small improvement for the old value head, but I wouldn't be surprised if its a stronger improvement for the new WDL head.

**Comment by Cyanogenoid** [comment]
Superseded by https://github.com/LeelaChessZero/lczero-training/pull/63.
